src/main/java/org/opensearch/security/dlic/rest/api/InternalUsersApiAction.java

```diff
@@ -335,7 +379,8 @@ public Map<String, RequestContentValidator.DataType> allowedKeys() {
                         return allowedKeys.put("backend_roles", DataType.ARRAY)
                             .put("attributes", DataType.OBJECT)
                             .put("description", DataType.STRING)
-                            .put("opendistro_security_roles", DataType.ARRAY)
+                            .put(OPENDISTRO_SECURITY_ROLES, DataType.ARRAY)
```

Comment:
Do we want either to be set or both should be allowed for now?

What is the plan to remove this deprecated key from allowed_keys?

Reply:
The plan is to mark these deprecated in 3.0, removal in 4.0. Hence, we need to keep both in 3.0 or decide on whether we just document in 3.0 and remove cleanly in 4.0